Dockerfile

```diff
@@ -1,23 +1,16 @@
 # Build image
-FROM golang:1.9-alpine AS build-env
+FROM golang:latest AS build-env
```

Comment:
I'm not sure about using `latest` images, at some point they can break the build and we are not even aware unless you periodically run the build.

Reply:
@ukjasonwright @TheColonelK what do you think? should we use versions to make sure our image is always stable?